WhyNotWin11.au3

```diff
@@ -460,7 +460,7 @@ Func Main()
 	GUICtrlSetColor(-1, 0xE20012)
 	GUICtrlSetFont(-1, $aFonts[$FontMedium] * _GDIPlus_GraphicsGetDPIRatio()[0])
 
-	GUICtrlCreateLabel("❌", 760, 10, 30, 30, $SS_CENTER + $SS_CENTERIMAGE)
+	GUICtrlCreateLabel("âťŚ", 760, 10, 30, 30, $SS_CENTER + $SS_CENTERIMAGE)
```

Comment:
I'm unsure why this is needed? Is this not displaying for some languages?

Reply:
It's not needed it is my mistake (wrong codepage). I would  replace it by `ChrW(0x274C)`.